scripts/travis/travis_install.sh

```diff
@@ -23,7 +23,7 @@ apt-get install \
 # Caffe requires a minimum CMake version of 2.8.8.
 if $WITH_CMAKE; then
   # cmake 3 will make sure that the python interpreter and libraries match
-  wget http://www.cmake.org/files/v3.2/cmake-3.2.3-Linux-x86_64.sh -O cmake3.sh
+  wget --no-check-certificate http://www.cmake.org/files/v3.2/cmake-3.2.3-Linux-x86_64.sh -O cmake3.sh
```

Comment:
This belongs in a separate commit, but I can verify that it's needed.

https://travis-ci.org/BVLC/caffe/jobs/88939834
https://travis-ci.org/BVLC/caffe/jobs/89066109
https://travis-ci.org/NVIDIA/caffe/jobs/89073960